com.unity.render-pipelines.universal/Editor/2D/Renderer2DMenus.cs

```diff
@@ -111,62 +101,62 @@ static bool CreateLightValidation()
             return Light2DEditorUtility.IsUsing2DRenderer();
         }
 
-        [MenuItem("GameObject/Light/Freeform Light 2D/Square", false, k_MenuPriority)]
+        [MenuItem("GameObject/Light/Freeform Light 2D/Square", priority = CoreUtils.Sections.section3 + CoreUtils.Priorities.gameObjectMenuPriority + 4)]
```

Comment:
why did we remove the 'false' (is Validate function) in this file? is not needed anymore because of the fact everything is handled nicely?

Reply:
We had  `false` because we need it to be able to call the ctor with the priority, but using `priority = ` on the argument of the constructor allow you to skip it, and it will get it's default value, that for `bool` in this case is false.